.rubocop.yml

```diff
@@ -5,6 +5,7 @@ AllCops:
     - spec/dummy/db/**/*
     - spec/dummy/config/**/*
     - spec/spec_helper.rb
+  TargetRubyVersion: 2.3.8
```

Comment:
Shouldn't we use something like 2.6.5?

Reply:
That's the oldest version that the CI uses